README.md

```diff
@@ -85,6 +88,12 @@ use {
 }
 ```
 
+### checkhealth
+
+Make sure you call `:checkhealth telescope` after installing telescope to ensure
+everything is setup correctly.
+
+
```

Comment:
I would put in a prominent link to the documentation somewhere in the getting started section, maybe at the end here. That way if people only read this section, they already know that there is extensive documentation available to them